src/System.Net.Http/src/System/Net/Http/Headers/KnownHeaders.cs

```diff
@@ -114,6 +114,22 @@ public StringAccessor(string s)
             public char CharAt(int index) => _string[index];
         }
 
+        // Can't use Span here as it's unsupported.
```

Comment:
BTW, this does seem like an artificial limitation, at least for this particular scenario (generic with a where constraint).